compliance/tags/tag_checker/tag_checker.pt

```diff
@@ -291,6 +291,17 @@ policy "untagged_resources_policy" do
   validate $filtered_resources do
     summary_template "Untagged Resources Summary"
     detail_template <<-EOS
+<<<<<<< HEAD
```

Comment:
Looks like you missed resolving a conflict.

Reply:
good catch.  thank you.